tox_poetry_installer/hooks.py

```diff
@@ -219,24 +133,28 @@ def tox_testenv_install_deps(venv: ToxVirtualEnv, action: ToxAction) -> Optional
         )
 
         env_deps = utilities.find_additional_deps(
-            packages, virtualenv, poetry, venv.envconfig.locked_deps
+            packages, virtualenv, poetry, tox_env.conf["locked_deps"]
         )
 
         logger.info(
             f"Identified {len(env_deps)} environment dependencies to install to env"
         )
 
         install_project_deps = (
-            venv.envconfig.install_project_deps
-            if venv.envconfig.install_project_deps is not None
-            else (
-                not venv.envconfig.skip_install and not venv.envconfig.config.skipsdist
-            )
+            tox_env.conf["install_project_deps"]
+            if tox_env.conf["install_project_deps"] is not None
+            else (not tox_env.conf["skip_install"] and not tox_env.core["no_package"])
```

Comment:
This logic can be cleaned up. Since the `_postprocess_install_project_deps()` function has been removed this config variable is no longer ternary and can be treated as a simple boolean.

All instances of the `install_project_deps` variable can be replaced with just `tox_env.conf["install_project_deps"]`

Reply:
removed the ternary and now simply checking the conf value directly